project/FrontendBuild.scala

```diff
@@ -27,7 +27,8 @@ object FrontendBuild extends Build with MicroService {
     "org.pegdown" % "pegdown" % "1.6.0" % scope,
     "org.jsoup" % "jsoup" % "1.8.1" % scope,
     "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
-    "org.scalatestplus.play" %% "scalatestplus-play" % "1.5.1" % scope
+    "org.scalatestplus.play" %% "scalatestplus-play" % "1.5.1" % scope,
+    "org.mockito" % "mockito-core" % "2.7.4" % scope
```

Comment:
I know. Ben upgraded agent-paye to 2.5.0 during the Play upgrade and since this is a new service I thought I'd try the latest version.